src/resources/storefront/transactions-resource.js

```diff
@@ -1,22 +1,22 @@
-export default function TransactionsResource({apiHandler}) {
-    return {
-        getAll({limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`transactions`, params);
-        },
-
-        get({id} = {}) {
-            return apiHandler.get(`transactions/${id}`);
-        },
+/**
+* This file was auto-generated by rebilly-sdk-generator.
+* Do not make direct changes to this file.
+*/
 
-        update({id, data} = {}) {
```

Comment:
I wonder why patch removed?

Reply:
StorefrontPatchTransaction is used by storefront-payment-form

Reply:
We were talking about that with Arif. This patch is deprecated and we should use `StorefrontPostPayment` instead. But 
I will manually add this update until do that migration.